src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/pooling_gpu_int8_ref.cl

```diff
@@ -30,7 +30,7 @@ KERNEL(pooling_gpu_int8_ref)(
 #endif
 )
 {
-#if OUTPUT_LAYOUT_BFYX  || OUTPUT_LAYOUT_BYXF || OUTPUT_LAYOUT_B_FS_YX_FSV4 || OUTPUT_LAYOUT_BFZYX
+#if OUTPUT_LAYOUT_BFYX  || OUTPUT_LAYOUT_BYXF || OUTPUT_LAYOUT_B_FS_YX_FSV4 || OUTPUT_LAYOUT_BFZYX || OUTPUT_LAYOUT_BS_FS_ZYX_BSV32_FSV32
```

Comment:
`bs_fs_zyx_bsv16_fsv16` is missing here.

Reply:
add it